oca_dependencies.txt

```diff
@@ -1,5 +1,5 @@
 partner-contact
 account-financial-reporting
 intrastat
-server-tools
+server-tools https://github.com/janverb/server-tools.git 10.0-use-acme-client
```

Comment:
I'll keep this as reminder that we can't merge before our dependency is merged and this changed